moss/src/registry/plugin/active.rs

```
@@ -24,7 +32,13 @@ impl Eq for Active {}
 impl Active {
     /// Return a new Active plugin for the given state + install database
     pub fn new(state: Option<State>, db: db::meta::Database) -> Self {
-        Self { state, db }
+        let cached_providers = db.all_providers().unwrap_or_default().into_iter().into_group_map();
```

Instead of `unwrap_or_default` which swallows any error, how about 

```rust
.unwrap_or_else(|e| {
    tracing::warn!(
        error = &e as &dyn std::error::Error,
        "failed to cache providers",
    );
    Default::default()
})
```

---

Actually, looks like we later _only_ check this cache - so really this should be called `load` [`_into_memory`], right? "cache" seems like the wrong name to me.